hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -30,6 +31,7 @@
     { name: "rxerr",       desc: "SDI in FwMode has error" },
     { name: "rxoverflow",  desc: "RX Async FIFO overflow" },
     { name: "txunderflow", desc: "TX Async FIFO underflow" },
+    { name: "tpm_cmdaddr_notempty", desc: "TPM Command/Address buffer available" }
```

Comment:
i left a question in the issue also, it's not super clear to me if hardware is supposed to detect command / address and come in to handshake, or only on the presence of write data for writes, and read command for reads. 

If it's only on command / address for both cases, wouldn't we need another interrupt to indicate that all write data is deposited, or that transaction has been de-asserted for the software to come back and check?

Reply:
It is not. I added a comment in the issue.

Reply:
sorry @eunchan, one more question about this interrupt. 
In the WRITE case, since the HW will automatically decode the address / command and start depositing data, do we want to interrupt on tpm_cmdaddr_notempty? Or should we wait for the entire payload to be done?

Reply:
I didnot put the write fifo interrupt intentionally. As TPM spec says, currently only 4B payload exists even the spec allows 64B payload. So, after receiving the `cmdaddr_notempty` , the core may have 32 cycles (in SCK) to wait.

I assume that the core needs to pop 4 entries for a command in most cases. One pop would take 3 or more core clock cycles + 2 or 3 peripheral clock cycles (@24MHz). So, I think the core has not much cycles left for the 32 SCK cycles.

Reply:
ah okay fair point. We'll just need to make a note of that in the spec. I think this is reasonable.